components/custom-button.tsx

```diff
@@ -0,0 +1,13 @@
+import { blueDarkAlt } from '@pxblue/react-native-themes';
+import React from 'react';
+import { Button, useTheme } from 'react-native-paper';
+
+export const MyCustomButton: typeof Button = (props) => {
+    const theme = useTheme();
```

Comment:
I don't know why GitHub is not letting me make suggestions...but:

```
const theme = useTheme(props.theme);
```
this will merge the theme provided via props (if any) with the theme from the closest theme provider.